coursier.bzl

```diff
@@ -866,6 +898,14 @@ pinned_coursier_fetch = repository_rule(
         "generate_compat_repositories": attr.bool(default = False),  # generate a compatible layer with repositories for each artifact
         "maven_install_json": attr.label(allow_single_file = True),
         "override_targets": attr.string_dict(default = {}),
+        "strict_visibility": attr.bool(
+            doc = """Controls visibility of transitive dependencies
+
+            If "True", transitive dependencies are private and invisible to user's rules.
+            If "False", transitive dependencies are public and visible to user's rules.
+            """,
+            default = True,
```

Comment:
Let's set this to False to prevent this from being a breaking change, and also be consistent with the macro.